src/Elewant/AppBundle/Resources/views/Layout/base.html.twig

```diff
@@ -41,6 +41,9 @@
 <script type="application/javascript" src="{{ asset('build/js/popper.min.js') }}"></script>
 <script type="application/javascript" src="{{ asset('build/js/bootstrap.min.js') }}"></script>
 
+<script src="{{ asset('bundles/fosjsrouting/js/router.js') }}"></script>
```

Comment:
I know this isn't needed (browsers assume javascript by default), but let's add `type="application/javascript"` to stay consistent with the `<script>` tags above and below 😉

Reply:
Yep, good catch. I copy-pasted this from the manual, like a proper n00b 👍